docs/setup/kubeedge_install_source.md

```diff
@@ -0,0 +1,165 @@
+# Setup from Source Code
+
+This guide provide steps which can be utilised to install KubeEdge Cloud and Edge side. At this point, we assume that you would have installed the [Pre-Requisite](kubeedge_install.md#prerequisite) for Cloud and Edge.
+
```

Comment:
I like to add the install of golang here.

Reply:
@subpathdev  We have mentioned to install the golang in the Pre-Requisite. Refer `At this point, we assume that you would have installed the [Pre-Requisite](kubeedge_install.md#prerequisite) for Cloud and Edge.`

Reply:
OK you have made changes in the other file. .... (can be closed)